networking/p2p/src/main/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/rpc/RpcMessageHandler.java

```diff
@@ -204,19 +205,20 @@ public void channelActive(ChannelHandlerContext ctx) {
     }
 
     @Override
-    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause)
-        throws IllegalArgumentException {
+    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
+      LOG.error("Unhandled error while processes req/response", cause);
       IllegalArgumentException exception = new IllegalArgumentException("Channel exception", cause);
```

Comment:
Should this be an `IllegalStateException`?

Reply:
Yes, done.